openrr-command/src/robot_command.rs

```diff
@@ -481,6 +482,26 @@ pub fn load_command_file_and_filter(file_path: PathBuf) -> Result<Vec<String>, O
         .collect())
 }
 
+fn filter_escape_double_quote(robot_command: &[String]) -> Vec<String> {
+    let mut concatenation = false;
+    let mut command_vector: Vec<String> = Vec::new();
+
+    for element in robot_command {
+        let command_element = element.replace('\"', "");
+        if concatenation {
+            let mut new_command = command_vector.pop().unwrap() + " ";
```

Comment:
I think it's more simple like below.

```rust
            let new_command = command_vector.pop().expect("parse double quote logic error");
            command_vector.push(format!("{new_command} {command_element}"));
```

Reply:
Thank you for comments. I fixed the areas indicated.